src/screens/SignIn/index.tsx

```diff
@@ -13,20 +19,38 @@ export const signInScreenTestIds = {
   passwordField: 'sign-in-form__input-password',
   forgotButton: 'sign-in-form__forgot-button',
   signInButton: 'sign-in-form__button',
+  errorAlert: 'sign-in__error-alert',
+  loadingDialog: 'sign-in__loading-dialog',
 };
 
 const SignInScreen = (): JSX.Element => {
```

Comment:
Will you add the UI test for this screen?

Reply:
@hoangmirs the UI test will be added in a separate task. I added it [here](https://github.com/manh-t/react-survey/pull/62) 🙏